notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<p>Is there a reason for using a more complicated bucket creation step?</p><p>Prefer to use the standard unless there's a good reason otherwise.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/726/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
I followed the other tabular workflows notebook (https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/automl/automl_tabular_on_vertex_pipelines.ipynb) and would like to keep it consistent. Could you point me to an example where we use the standard way? I can test and update it for both notebooks in a follow up.

Reply:
<p>Nevermind, we plan to remove bucket creation in the future.</p>